### PR TITLE
Removed React-Modal package and fixed new modal issue

### PR DESCRIPTION
**Pull request by ascode94**
Removed React-Modal package and fixed new modal issue

### CODE REVIEW

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
LGTM, Thank you @ascode94 ,
Just fix the typo.
also please ask @MoeNick  about the vertical scroll when the modal is open.
we hide the vertical scroll of the window if the modal is open.

File: src/components/modals/Modal.tsx [hunk 1]
```diff
@@ -100,21 +110,23 @@ const Background = styled.div`
...
-const ModalWrapper = styled.div`
+const ModalWrapper = styled.div<ModalWrapperProps>`
 	background-color: ${props =>
 		props.theme === ETheme.Dark
 			? brandColors.giv[600]
 			: neutralColors.gray[100]};
 	box-shadow: 0 3px 20px
 		${props => (props.theme === ETheme.Dark ? '#00000026' : '#21203c')};
-	border-radius: 8px;
+	border-radius: ${props => (props.fullScreen ? '0px' : '8px')}
```
Comment (MohammadPCh) [inline src/components/modals/Modal.tsx:120]: please add a semicolon at the end of the line.

File: src/components/modals/WelcomeModal.tsx [hunk 2]
```diff
@@ -35,13 +35,13 @@ const WelcomeModal = ({ setShowModal }: ISignInModal) => {
...
-			<CloseButton onClick={closeModal}>
-				<Image src={closeIcon} alt='close' />
-			</CloseButton>
+		<Modal showModal setShowModal={setShowModal} fullScreen hideHeader>
 			<ModalGrid>
 				<BGContainer />
 				<ContentContainer>
+					<CloseButton onClick={closeModal}>
+						<Image src={closeIcon} alt='close' />
+					</CloseButton>
```
Comment (MohammadPCh) [inline src/components/modals/WelcomeModal.tsx:44]: It's better to move the close button to the Modal component.

**Review by MohammadPCh — CHANGES_REQUESTED** [review]

File: src/components/modals/WelcomeModal.tsx (hunk 2 above)
Comment (ascode94) [reply src/components/modals/WelcomeModal.tsx]: @MohammadPCh Thanks for the review. I made the changes you mentioned.

**Review by MohammadPCh — APPROVED** [review]
Thank You @ascode94 for adding the full-screen feature to the modal component.
It's great.

**Pull request merged**